### PR TITLE
Clarify exceptions to task selection

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

**Review by donat — APPROVED** [review]

File: subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc [hunk 1]
```diff
@@ -66,17 +66,26 @@ This will run the single "myTask" and also all of its <<tutorial_using_tasks.ado
...
-You can also run a task for _all_ subprojects by again using a task _selector_ that consists of the task name only. For example, this will run the "test" task for all subprojects when invoked from the root project directory:
+You can also run a task for _all_ subprojects by again using a task _selector_ that consists of the task name only.
+For example, this will run the "test" task for all subprojects when invoked from the root project directory:
 ----
 $ gradle test
 ----
 
+[NOTE]
+====
+Some tasks selectors, like `help`, will only run the task on the project they are invoked on and not on all the subprojects.
```
Comment (donat) [inline subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc:85]: Maybe we should list _all_ core tasks that have special behavior.

> Some tasks selectors - like `help`, `dependencies`, and `dependencyInsight` - will only run the task on the project they are invoked on and not on all the subprojects.

**Comment by donat** [comment]
@bot-gradle test and merge

File: subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc (hunk 1 above)
Comment (ljacomet) [reply subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc:85]: The whole list is longer than that ... it even has tasks I have no idea what they are, like the `ModelReport` ...
So I went with the short version.

Comment (donat) [reply subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc:85]: 👍

**Pull request merged**